README.md

```diff
@@ -0,0 +1,52 @@
+# Tech Docs Template
+
+The Tech Docs Template is a static site generator to build technical documentation with a GOV.UK style for GaaP (Government as a Platform) products.
+
+## Getting started
+
+To preview or build the website, we need to use the terminal.
```

Comment:
I find the use of `we` throughout the documentation a bit jarring. Not really sure why, I just think it'd make more sense to use 'you' throughout, e.g.

"To preview or build the website, you'll need to use the terminal."